app/controllers/users/sessions_controller.rb

```diff
@@ -206,6 +207,32 @@ def sign_in_params
     end
   end
 
+  def check_password_compromised
+    return if current_user.check_password_compromised_at.present? && 
+      eligible_for_password_lookup?
+
+    user_session[:redirect_to_phone_compromised] = PwnedPasswords::LookupPassword.call(auth_params[:password])
```

Comment:
as long as we're here, what if we renamed `PwnedPasswords::LookupPassword.call` to something like `exists?` so it's clearer that it returns a true/false?

Reply:
Good idea. I think exists makes sense